src/server/pkg/obj/local_client.go

```diff
@@ -26,8 +26,16 @@ type localClient struct {
 	root string
 }
 
+func (c *localClient) normPath(path string) string {
```

Comment:
The local storage client seemed to be used in a couple different ways, one of which was problematic on windows (but works fine in linux).  Specifically, it appears that we duplicated the path by both configuring a path in the client and passing the same path for all keys.  In windows this breaks for absolute paths that include a drive letter because the `:` is invalid in filenames.

Now, we just sandbox the path to the configured local storage root.